nukebuild/Build.cs

```diff
@@ -123,8 +123,18 @@ void ExecWait(string preamble, string command, string args)
         EnsureCleanDirectory(Parameters.TestResultsRoot);
     });
 
+    Target CompileNative => _ => _
+    .DependsOn(Clean)
+    .OnlyWhen(() => EnvironmentInfo.IsOsx)
+    .Executes(() =>
+    {
+        var project = $"{RootDirectory}/native/Avalonia.Native/src/OSX/Avalonia.Native.OSX.xcodeproj/";
+        var args = $"-project {project} -configuration {Parameters.Configuration} CONFIGURATION_BUILD_DIR={RootDirectory}/Build/Products/Release";
```

Comment:
Are you sure that it is ignored by `.gitignore`?

Reply:
i haven't touched or checked anything related to `.gitignore` but it would make sense to add this dir to` .gitignore` eventually